packages/component-model-manuscript/entities/manuscript/index.js

```diff
@@ -143,6 +143,22 @@ class Manuscript extends BaseModel {
     return manuscript
   }
 
+  static async findById(id, related = '[teams, files]') {
+    // This is a copypasta of `find` because `find` might be changed so that it doesn't query by userId
+    // TODO: Once we've got the policies in place, replace `find` with this function
+    const [manuscript] = await this.query().where({
```

Comment:
This can use objections `findById` query builder function https://vincit.github.io/objection.js/api/query-builder/find-methods.html#findbyid